enemies/arrowtrap/arrow_trap.gd

```diff
@@ -0,0 +1,25 @@
+extends Entity
+
+export (float) var timer
+export (String) var direction
+var movetimer = 0
+
+func _init():
+	TYPE = "TRAP"
+
+func _ready():
+	spritedir = direction
+	if spritedir == "Left":
+		$AnimatedSprite.set_animation(spritedir.to_lower())
+	if spritedir == "Up":
+		$AnimatedSprite.set_animation(spritedir.to_lower())
+	if spritedir == "Right":
+		$AnimatedSprite.set_animation(spritedir.to_lower())
+	if spritedir == "Down":
+		$AnimatedSprite.set_animation(spritedir.to_lower())
```

Comment:
Use instead

if ["Left", "Up", "Right", "Down"].has(spritedir):
  $AnimatedSprite.set_animation(spritedir.to_lower())

Reply:
I wholeheartedly agree and updated it. Thank you!